var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -75,14 +81,14 @@ class RocmOpenmpExtras(Package):
     depends_on('elfutils', type=('build', 'link'))
     depends_on('libffi', type=('build', 'link'))
 
-    for ver in ['3.9.0', '3.10.0', '4.0.0']:
+    for ver in ['3.9.0', '3.10.0', '4.0.0', '4.1.0']:
         depends_on('hsakmt-roct@' + ver, type=('build', 'run'), when='@' + ver)
         depends_on('comgr@' + ver, type='build', when='@' + ver)
         depends_on('hsa-rocr-dev@' + ver, type=('build', 'run'),
                    when='@' + ver)
         depends_on('rocm-device-libs@' + ver, type=('build', 'run'),
                    when='@' + ver)
-        depends_on('llvm-amdgpu@' + ver, type=('build', 'run'),
+        depends_on('llvm-amdgpu@' + ver + ' -openmp', type=('build', 'run'),
```

Comment:
```suggestion
        depends_on('llvm-amdgpu@' + ver + ' ~openmp', type=('build', 'run'),
```

Reply:
done

Reply:
@haampie @estewart08 Anything pending here?. 
Just want to make sure we get this in, as this is only recipe left for rocm 4.1 release.

Reply:
ping

Reply:
seems @estewart08 has addressed the comments from @haampie .... 
@tldahlgren can you help approve/merge this ?